app/utils/decorator_utils.py

```diff
@@ -0,0 +1,11 @@
+from app.database.models.user import UserModel
+
+def check_mail_confirmation(f):
```

Comment:
Please replace f with a suitable informative argument/variable name.

Reply:
Check it now.